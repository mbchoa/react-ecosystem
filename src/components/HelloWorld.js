import React from 'react';
import withStyles from 'isomorphic-style-loader/lib/withStyles';

import s from './HelloWorld.css';

export const HelloWorld = (props, context) =>
    <div className={s.hello}>
        Hello from React Ecosystem!
    </div>

export default withStyles(s)(HelloWorld);
