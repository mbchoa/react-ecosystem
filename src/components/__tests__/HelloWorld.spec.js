import React from 'react';
import { shallow } from 'enzyme';

import { HelloWorld } from '../HelloWorld';

describe('HelloWorld', () => {
    function renderComponent() {
        return shallow(<HelloWorld />);
    }

    it('should render HelloWorld', () => {
        expect(renderComponent()).toMatchSnapshot();
    })
});
