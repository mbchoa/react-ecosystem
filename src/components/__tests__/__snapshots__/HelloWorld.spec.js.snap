// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld should render HelloWorld 1`] = `
<div
  className="hello"
>
  Hello from React Ecosystem!
</div>
`;
